mne/viz/tests/test_topomap.py

```diff
@@ -242,7 +242,7 @@ def test_plot_topomap_basic():
     fig, _ = plot_topomap(data, info_grad)
 
     # check that channels are scattered around x == 0
-    pos = img.axes.collections[-1].get_offsets()
+    pos = fig.axes.collections[-1].get_offsets()
```

Comment:
@larsoner Well, technically, this is actually not `fig`, but `img`, I read @agramfort suggestion too quickly. :)